.github/workflows/coverity.yml

```diff
@@ -0,0 +1,25 @@
+name: Coverity Scan
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Once every day at 00:00 UTC
+env:
+  CC: gcc
+  ARCH: x86_64
+  CONTAINER_NAMESPACE: ghcr.io/opendataplane/odp-docker-images
+  OS: ubuntu_20.04
+  COVERITY_EMAIL: odp@lists.opendataplane.org
+  COVERITY_PROJECT: odp
```

Comment:
Did you test that this project name works? I'm thinking the project name may need to be changed to `ODP` like in the odp-docker-images repo.

Reply:
Since I don't have the token of ODP (and thus couldn't test it), I've tested with https://scan.coverity.com/projects/jlinkes-odp. I built the image locally (with jlinkes-odp token) and tried testing with that image in github actions.

I'll change the repo name when we get a passsing build in 
odp-docker-images.

Reply:
Okay, the docker image is now built and pushed to the container registry (https://github.com/orgs/OpenDataPlane/packages/container/package/odp-docker-images%2Fodp-ci-ubuntu_20.04-x86_64-coverity-linux-generic).